crates/bevy_scene/src/dynamic_scene_builder.rs

```diff
@@ -131,6 +140,45 @@ impl<'w> DynamicSceneBuilder<'w> {
         drop(type_registry);
         self
     }
+
+    /// Extract resources from the builder's [`World`].
+    ///
+    /// Only resources registered in the builder's [`AppTypeRegistry`] will be extracted.
+    /// ```
+    /// # use bevy_scene::DynamicSceneBuilder;
+    /// # use bevy_app::AppTypeRegistry;
+    /// # use bevy_ecs::prelude::{ReflectResource, Resource, World};
+    /// # use bevy_reflect::Reflect;
+    /// #[derive(Resource, Default, Reflect)]
+    /// #[reflect(Resource)]
+    /// struct MyResource;
+    ///
+    /// # let mut world = World::default();
+    /// # world.init_resource::<AppTypeRegistry>();
+    /// let _resource = world.insert_resource(MyResource);
```

Comment:
Is this line intentional? `World::insert_resource` doesn't return anything.
```suggestion
    /// world.insert_resource(MyResource);
```

Reply:
Nope, my bad!